ui/src/parseError.ts

```diff
@@ -68,6 +68,10 @@ export function parseError(error: any): IParsedError {
 
     message = parseIfHtml(message);
 
+    // Azure storage SDK errors are presented in XML
+    // https://github.com/Azure/azure-sdk-for-js/issues/6927
+    message = parseIfXml(message);
```

Comment:
The original location for this worked but it made more sense to put it next to `parseIfHtml`